frontend/benefit/applicant/src/components/mainIngress/styled.ts

```diff
@@ -0,0 +1,57 @@
+import { Button } from 'hds-react';
+import React from 'react';
+import { Theme } from 'shared/styles/theme';
+import styled from 'styled-components';
+
+const StyledContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: ${props => (props.theme as Theme).spacing.m};
```

Comment:
nitpick: A little bit typesafer way to do this that avoids type assertion:
```
import styled, { ThemeProps} from 'styled-components';     
type Props = ThemeProps<Theme>; // this could be exported from some d.ts file as ThemeProps or something
...
 padding-bottom: ${(props: Props) => props.theme.spacing.m}; 
```

Reply:
does not work like that

Reply:
Hmm... at least it compiled for me?